README.md

```diff
@@ -1,2 +1,63 @@
-# hello_microverse
-the first project in microverse
+# hello microverse
+
+> Description the project.
+
+
+## Built With
+
+- html&css
+- no Frameworks
+- linter
+
+
+## Getting Started
+
+**You can add the steps to follow to clone your project,
+
+Step 1:if you dont have git installed on your machine you cad download it from [here](https://git-scm.com/downloads).
+
+Once you have git installed on your machine you can clone your project by running the command below to clone your project to your local machine
+
+
+`git clone <your project link>`
+
+Alternatively you can download the zip file of your project by clicking on the `Code` button on the right side of your project page and clicking on `Download ZIP`
+
+Step 2: Locate the folder where you cloned your project to and open the `index.html` file in your browser to view your project.**
+
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] Please kindly personalized this statement. You are almost there! You need to specify the link to your repository. https://github.com/ilyassatlassi/hello_microverse

Reply:
@OyePriscilla please can you give more details